### PR TITLE
Helper to add new input on lstm network

### PR DESCRIPTION
**Pull request by mikefaille**

### CODE REVIEW

File: data_manipulation.py [hunk 1]
```diff
@@ -1,42 +1,46 @@
...
-
+    close_scaler = preprocessing.MinMaxScaler()
     close = close_scaler.fit_transform(close)
+    # close_p_scaler = preprocessing.MinMaxScaler()
+    # close_p = (close[1:] - close[:-1]) / close[1:]
+    # close_p = close_p_scaler.fit_transform(close_p)
+
+    extra_input_list = []
+    extra_scaler_list = []
+    if len(extra_input_list_str) != 0:
```
Comment (benoxoft) [inline data_manipulation.py]: checking that the list is empty before doing a for loop is not necessary

File: data_manipulation.py [hunk 2]
```diff
@@ -1,42 +1,46 @@
...
+        for input in extra_input_list_str:
+            extra_input, extra_scaler = generate_input_data(train_data, input)
+            extra_input_list = extra_input_list + [extra_input[1:]]
+            extra_input_scaler = extra_scaler_list + [extra_scaler]
 
-    fcf = train_data['fcf'].values.astype(float)
-    fcf = np.array(fcf).reshape((len(fcf), 1))
-    fcf = fcf_scaler.fit_transform(fcf)
+    data_to_concat = tuple(extra_input_list)
```
Comment (benoxoft) [inline data_manipulation.py:41]: I don't think you have to convert your list into a tuple

File: data_manipulation.py [hunk 3]
```diff
@@ -1,42 +1,46 @@
...
+    data = np.concatenate((data_to_concat), axis=1)
```
Comment (benoxoft) [inline data_manipulation.py]: parenthesis are redundant

**Review by benoxoft — APPROVED** [review]
👍

File: data_manipulation.py (hunk 1 above)
Comment (mikefaille) [reply data_manipulation.py]: Fix en retrouvant si la liste est déclaré.

File: data_manipulation.py (hunk 2 above)
Comment (mikefaille) [reply data_manipulation.py:41]: It should be ok after discussion

**Pull request merged**